### ISSUE TITLE
MUC - invites appear to come from MUC name part

### ISSUE THREAD

**Issue by licaon-kter** [conversejs/converse.js#1836]
...instead of user.

HEAD (d17ae811d758e35f24770bd46ad88ee976f5c6b3), Firefox 72

### Steps:
* Conversations -> New group/channel
* choose Converse user

### Expected:
* popup with `romeo@domain.tld has invited you to join a groupchat: myhamlet@conference.somedomain.tld`

### Actual result:
* popup with `myhamlet has invited you to join a groupchat: myhamlet@conference.somedomain.tld`

**Comment by licaon-kter** [comment]
Umm now I see `myhamlet@conference.somedomain.tld invited you to myhamlet@conference.somedomain.tld` which still does not sound right as I expect _(better said find more useful)_ to be invited by a user not by the MUC itself _(specially when MUC ids are randomly generated)_

iNPUTmice says: _"you might have reported that against mediated invites, but that fix is for direct invites"_

**Comment by iNPUTmice** [comment]
The "problem" is that ejabberd sends a mediated and a direct invite in the same message stanza. As far as I can tell Converse.js doesn’t support mediated invites and only parses the direct invite.
Direct invites _usually_ come from the the inviter. Therefor Converse.js correctly parses the from of the message stanza - which in this case is the room itself.

**Comment by jcbrand** [comment]
You're right @iNPUTmice. @licaon-kter can you post an example stanza from ejabberd so that I can write a test for it?

**Comment by iNPUTmice** [comment]
```xml
<message xmlns="jabber:client" from="conversations@conference.siacs.eu" to="christ19@jabber.de">
  <x xmlns="http://jabber.org/protocol/muc#user">
    <invite xmlns="http://jabber.org/protocol/muc#user" from="daniel@example.com/Conversations.QS-P"/>
  </x>
  <x jid="conversations@conference.siacs.eu" xmlns="jabber:x:conference"/>
  <body xml:lang="en" xmlns="jabber:client">daniel@example.com/Conversations.QS-P invites you to the room conversations@conference.siacs.eu</body>
</message>
```

**Comment by licaon-kter** [comment]
@jcbrand no, not really :))

Do I see this in the debug console?
